internal/handlers/pg/msg_findandmodify.go

```diff
@@ -310,6 +317,11 @@ func prepareFindAndModifyParams(document *types.Document) (*findAndModifyParams,
 		return nil, err
 	}
 
+	maxTimeMS, err := common.GetPositiveNumber(document, "maxTimeMS")
+	if err != nil {
+		return nil, err
```

Comment:
Hey, @DoodgeMatvey  )
can you please add at least one test for `find and modify` to cover this `if` branch? Thank you.

Reply:
Added